bundles/org.openhab.ui/web/src/pages/developer/log-viewer.vue

```diff
@@ -225,12 +225,27 @@
   td.sticky
     position sticky
     left 0
+    width 105px
     color black
     background #f1f1f1
     z-index 1
     white-space nowrap
     overflow hidden
 
+  td.level
+    width 50px
+
+  td.logger
+    width 280px
```

Comment:
I didn’t need this when testing in Firefox. It will not harm either.

Reply:
I needed this when developing in Chromium.

Reply:
For some strange reason, when I was trying this, things worked in Firefox, but I couldn’t even get the websocket connection to start in Chrome, so no way to test (there was an error in the console). That is obviously another issue and I will keep an eye out for it.